docs/Server.md

```diff
@@ -850,7 +850,7 @@ fastify.setErrorHandler(function (error, request, reply) {
 })
 ```
 
-Fastify is provided with a default function that is called if no error handler is set and that logs the error with respect to its `statusCode`:
+Fastify is provided with a default function that is called if no error handler is set. It can be accessed using `fastify.errorHandler` and it logs the error with respect to its `statusCode`.
```

Comment:
The `errorHandler` property should have its own entry in the `Server.md` (similar to `defaultTextParser`).